test/transcoding/OpConstantSampler.ll

```diff
@@ -20,8 +20,8 @@
 
 ; CHECK-SPIRV: ConstantSampler {{[0-9]+}} [[SamplerID0:[0-9]+]] 3 1 0
 ; CHECK-SPIRV: ConstantSampler {{[0-9]+}} [[SamplerID1:[0-9]+]] 0 0 0
-; CHECK-SPIRV: SampledImage {{.*}} [[SamplerID0]]
-; CHECK-SPIRV: SampledImage {{.*}} [[SamplerID1]]
+; CHECK-SPIRV: SampledImage {{.*}} {{.*}} {{.*}} [[SamplerID0]]
```

Comment:
`{{.*}} -> {{[0-9]+}}`
Please, apply to all other instances of `{{.*}}` next to each other.

Reply:
Fixed, checked.